src/components/general/footer/Footer.tsx

```diff
@@ -1,93 +1,50 @@
 /* Import Dependencies */
-import { useState } from "react";
 import { useLocation } from "react-router-dom";
-import classNames from 'classnames';
+import classNames from "classnames";
 import { Container, Row, Col } from "react-bootstrap";
 
 /* Import Styles */
-import './footer.scss';
-
-/* Import Icons */
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faChevronUp } from "@fortawesome/free-solid-svg-icons";
+import styles from './footer.module.scss';
```

Comment:
Shouldn't this also be double quotes?

Reply:
In principle, it does not matter, but overall I decided to use single quotes for import statements.
Some of the older files still got the double quote imports, will replace any I find with single quotes.